src/gql/queries/get-task.graphql

```diff
@@ -2,112 +2,112 @@
 #import "../fragments/annotation.graphql"
 
 query GetTask($taskId: String!, $execution: Int) {
```

Comment:
Assuming this diff is just reorganizing fields and the fields are the same lgtm.